src/chart/pie/PieView.ts

```diff
@@ -242,6 +244,28 @@ class PieView extends ChartView {
             }
         }
 
+        // remove empty-circle if it exists
+        if (this._emptyCircleSector) {
+            group.remove(this._emptyCircleSector);
+        }
+        // when all data are filtered, show lightgray empty circle
+        if (data.count() === 0 && seriesModel.get('showEmptyCircle')) {
+            const {cx, cy, r0, r } = getBasicPieLayout(seriesModel, api);
+            const sector = new graphic.Sector({
+                shape: {
+                    cx,
+                    cy,
+                    r0,
+                    r,
+                    startAngle: 0,
+                    endAngle: Math.PI * 2
+                }
+            });
```

Comment:
The sector has a default value for `startAngle` and `endAngle`, so this can be simplified as
```js
const sector = new graphic.Sector({
    shape: getBasicPieLayout(seriesModel, api)
});
```

Reply:
fixed